### PR TITLE
SEO-176062-Windows-forms-duplicate-title

### PR DESCRIPTION
**Pull request by Edithabich**
HI @MallikaRK ,
Have updated UG titles please review.
Thanks.

Regards,
Edith Abich.

### CODE REVIEW

File: WindowsForms/Classic/Month-Calendar/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms xptoolbar control | Syncfusion
+title: Overview of Windows Forms Month Calendar control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Classic/Month-Calendar/Overview.md:3]: Overview of Windows Forms CalendarDateTime Control | Syncfusion

File: WindowsForms/Calculation-Engine/Overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms Calculation Engine control | Syncfusion
+title: Overview of Windows Forms Calculation Engine control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Calculation-Engine/Overview.md:3]: C under Control should be in caps. Check in all places.

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Classic/Month-Calendar/Overview.md (hunk 1 above)
Comment (Edithabich) [reply WindowsForms/Classic/Month-Calendar/Overview.md]: resolved

File: WindowsForms/ComboBox-DropDown/Overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Overview of Syncfusion ComboDropDown for WindowsForms
+title: Overview of Windows Forms ComboBox DropDown Control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/ComboBox-DropDown/Overview.md:3]: Use ComboDropDown

File: WindowsForms/Overview.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Overview of Syncfusion Essential Studio Windows Forms products
+title: Overview of Windows Forms | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Overview.md:3]: Overview of Essential Studio Windows Forms products | Syncfusion

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/ComboBox-DropDown/Overview.md (hunk 3 above)
Comment (Edithabich) [reply WindowsForms/ComboBox-DropDown/Overview.md]: resolved

File: WindowsForms/Overview.md (hunk 4 above)
Comment (Edithabich) [reply WindowsForms/Overview.md]: resolved

**Pull request closed without merging**